### PR TITLE
Import Queue consistently

### PR DESCRIPTION
**Pull request by MaxGiting**

### CODE REVIEW

File: queues.md [hunk 1]
```diff
@@ -449,9 +449,9 @@ If you would like to register an event that will be called when a queued job fai
 
     namespace App\Providers;
 
-    use Queue;
-    use Illuminate\Queue\Events\JobFailed;
+    use Illuminate\Support\Facades\Queue;
     use Illuminate\Support\ServiceProvider;
+    use Illuminate\Queue\Events\JobFailed;
```
Comment (GrahamCampbell) [inline queues.md]: Please keep this in the order it was. We sort by length. :)

File: queues.md [hunk 2]
```diff
@@ -449,9 +449,9 @@ If you would like to register an event that will be called when a queued job fai
 
     namespace App\Providers;
 
-    use Queue;
```
Comment (GrahamCampbell) [inline queues.md]: 👍

File: queues.md (hunk 1 above)
Comment (MaxGiting) [inline queues.md]: Thanks for the feedback. Now I know 👍

**Comment by GrahamCampbell** [comment]
👍

**Comment by taylorotwell** [comment]
Use 5.3 branch.

**Comment by MaxGiting** [comment]
This is not an issue on 5.3 branch 👍

**Comment by GrahamCampbell** [comment]
Oh, that's ok then. It's only a matter of time until a 5.3 -> master merge will happen.

**Pull request closed without merging**